### ISSUE TITLE
Custom trigger button

### ISSUE THREAD

**Issue by jenpope** [CuppaLabs/angular2-multiselect-dropdown#339]
Is it possible to do a custom trigger for multiselect (i.e., have a button trigger the dropdown)?

Similar to "Custom trigger element using transclusion" from here?

http://dotansimha.github.io/angularjs-dropdown-multiselect/docs/#/main

**Comment by CuppaLabs** [comment]
Yes, we have this feature. Refer to demo examples. Methods example section
for this. Let me know if this helps.

**Comment by jenpope** [comment]
Thank you! What is the remote trigger call?

I saw the open button, but did not see what the method being called was.

https://cuppalabs.github.io/angular2-multiselect-dropdown/#/dropdownMethods

Also, is it possible to hide the select field entirely and just have the dropdown body show up under the trigger button?  Like this?

https://i.stack.imgur.com/DNJMY.png

**Comment by chrisbratherton** [comment]
+1 

Documentation needs updating to show method calls for Open/Close. Current example only shows reset functionality.

**Comment by CuppaLabs** [comment]
please empty your browser cache and check example https://cuppalabs.github.io/angular2-multiselect-dropdown/#/dropdownMethods

![image](https://user-images.githubusercontent.com/3957664/51759000-32046e80-20ed-11e9-8856-57646c7c89b1.png)

**Comment by chrisbratherton** [comment]
Still doesn't show you how to implement the 'Open' button in the code example on that page.

**Comment by EduardRakov** [comment]
@chrisbratherton @jenpope see here 
https://github.com/CuppaLabs/angular2-multiselect-dropdown/blob/master/src/app/examples/resetdropdown.ts

**Comment by iTexphere** [comment]
> @chrisbratherton @jenpope see here
> https://github.com/CuppaLabs/angular2-multiselect-dropdown/blob/master/src/app/examples/resetdropdown.ts
> 
> @CuppaLabs the docs really should be updated

is this really work?
this.dropdownElem.openDropdown(); openDropdown() is not available
